projects/ng-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@indice/ng-components",
-  "version": "0.2.21",
+  "version": "0.2.3",
```

Comment:
Take some time and think about versioning. This is actually downgrading the version. This is lower than the previous `0.2.21`. In case of feature updates you should bump the frature segment in scemantic versioning. That would be 0.3.0. 

Also when rolling out a new features in parrallel one could use a version suffix **beta/alpha** 
This allows for the main version to stay the same under. This `0.3.0-beta.1` tells the user "We are actually preparing the 0.3.0 feature release with new features (duh) but we are not stable yet. Or even the radical `0.3.0-modals.1` according to the feature being tested. In the end the versioning progression stays consinstent and when `0.3.0` final is released will contain everything we want without skipping numbers for beta testing.

see here
https://kevinkreuzer.medium.com/publishing-a-beta-or-alpha-version-to-npm-46035b630dd7
https://gist.github.com/adcreare/00a10f389841dd4c97197608c9b33a7f

Reply:
Yes the version number was totally wrong. Point taken about versioning